templates/vc10.mpd

```diff
@@ -63,6 +65,7 @@
 <%foreach(configurations)%>
   <ImportGroup Condition="'$(Configuration)|$(Platform)'=='<%configuration%>|<%platform%>'" Label="PropertySheets">
     <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
+	<%if(custom_prop)%><Import Project="<%custom_prop%>" /><%endif%>
```

Comment:
Indent

Reply:
sorry, I haven't payed attention to the indent, but it looks like the indent is not affect the result.